src/components/inputs/radio-input/radio-option.js

```diff
@@ -68,11 +68,23 @@ Option.propTypes = {
     PropTypes.func,
   ]).isRequired,
   components: PropTypes.shape({
-    WrapperComponent: (props, propName) => {
-      if (props[propName] && !isValidElementType(props[propName])) {
-        return new Error(
-          `Invalid prop 'components.WrapperComponent' supplied to 'RadioInput.Option': the prop is not a valid React component`
-        );
+    wrapper: (props, propName) => {
```

Comment:
Wondering. Is this almost the same as `PropTypes.oneOf([PropTypes.func, PropTypes.array, PropTypes.elementType])`?

Reply:
Tried to restrict it to a function (1st if) that receives one argument (2nd if) and returns a valid react.element (3rd if)

Reply:
Maybe a bit over engineered and fails stupidly, thought it could help/guide the user maybe

Reply:
True. It's a nice idea! Maybe it's also something UIKit could share and reuse internally.